src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -2477,6 +2522,42 @@
         </TextBlock>
     </DataTemplate>
     
+
+    <Style TargetType="Expander" x:Key="InstalledPackagesExpanderStyle">
+        <Setter Property="Template">
+            <Setter.Value>
+                <!-- Control template for expander -->
+                <ControlTemplate TargetType="Expander" x:Name="exp">
+                    <Grid>
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto"/>
+                            <RowDefinition Name="ContentRow" Height="0"/>
+                        </Grid.RowDefinitions>
+                        <Border Name="Border" Grid.Row="0" Background="{x:Null}" BorderBrush="{x:Null}" BorderThickness="1" CornerRadius="0,0,0,0">
```

Comment:
whats up with null color, is this an invisible border?

Reply:
We stole this pattern from one of the other expanders. There is a subtle difference between setting a Brush to null or to Transparent and I just assume this is needed here.

Reply:
https://stackoverflow.com/questions/5344699/xnull-vs-transparent